recipes/unidist/meta.yaml

```diff
@@ -0,0 +1,131 @@
+{% set name = "unidist" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5b2cc602604f75ed6a9323cd4cd0585a1c806d3465f50ecb7ff31772b80f2edb
+
+build:
+  # noarch disabled because the recipe now has selector for Ray, see below
+  # noarch: python
+  number: 0
+  skip: true  # [py<37]
```

Comment:
```suggestion
```

Reply:
What will happen in this case if the user has python version < 37?

Reply:
You will need to add a minimum version wherever python is a dependency. Something like `python >=3.7`.

Reply:
I see, added python version deps.